charts/radarr/README.md

```diff
@@ -1,131 +1,71 @@
-# radarr movie download client
+# Radarr | A fork of Sonarr to work with movies à la Couchpotato
+Umbrella chart that
+* Uses [media-common](https://github.com/k8s-at-home/charts/tree/master/charts/media-common) as a base
+* Adds docker image information leveraging the [Linuxserver.io image](https://hub.docker.com/r/linuxserver/radarr/)
+* Deploys [Radarr](https://github.com/Radarr/Radarr)
 
-This is a helm chart for [radarr](https://github.com/Radarr/Radarr/) leveraging the [Linuxserver.io image](https://hub.docker.com/r/linuxserver/radarr/)
-
-## TL;DR;
-
-```shell
+## TL;DR
+```console
 $ helm repo add k8s-at-home https://k8s-at-home.com/charts/
 $ helm install k8s-at-home/radarr
 ```
 
 ## Installing the Chart
-
-To install the chart with the release name `my-release`:
-
+To install the chart with the release name `radarr`:
 ```console
-helm install --name my-release k8s-at-home/radarr
+helm install radarr k8s-at-home/radarr
 ```
 
 ## Upgrading
+Chart versions before 6.0.0 did not use media-common. Upgrading will require you to nest your values.yaml file under
+a top-level `radarr:` key.
 
-Chart versions 3.2.0 and earlier used separate PVCs for Downloads and Movies. This presented an issue where Radarr would be unable to hard-link files between the /downloads and /movies directories when importing media. This is caused because each PVC is exposed to the pod as a separate filesystem. This resulted in Radarr copying files rather than linking; using additional storage without the user's knowledge.
+Chart versions 3.2.0 and earlier used separate PVCs for Downloads and Music. This presented an issue where Radarr would be unable to hard-link files between the /downloads and /music directories when importing media. This is caused because each PVC is exposed to the pod as a separate filesystem. This resulted in Radarr copying files rather than linking; using additional storage without the user's knowledge.
```

Comment:
This may be because of copy/paste from lidarr, but I believe the references to 'Music' here should be 'Movies'?